__tests__/app.test.js

```diff
@@ -4,104 +4,138 @@ const data = require("../db/data/test-data");
+            
             expect(status).toBe(400);
             expect(message).toBe("banana is an invalid article_id (number)");
         });
     });
-    describe.only("/articles", () => {
-        test("200:status code and responds with article array of article objects and corresponding properties", async () => {
-            const { status, body } = await request(app).get("/api/articles");
-            const { articles } = body;
-            const expectedArticleProperties = ["author","title","article_id","topic","created_at",
-                "votes","article_img_url","comment_count"]
+    describe("/articles/:article_id/comments", () => {
+        test("200: status code responds with an array of comments for the given article_id", async () => {
+            const { status, body:{comments} } = await request(app).get("/api/articles/1/comments");
+            
+            const expectedCommentsTypes = {
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                article_id: expect.any(Number)
+            };
+            expect(status).toBe(200);
+            expect(comments.length).toBe(11);
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject(expectedCommentsTypes);
+            });
             
-            expect(status).toBe(200)
-            expect(articles.length).not.toBe(0)
-            articles.forEach((article) => {
-                expect(Object.keys(article)).toEqual(expectedArticleProperties)
-            })
-        
-        });
-        test("200: articles are sorted by created_at date in descending order",async() => {
-            const { body } = await request(app).get("/api/articles")
-            const { articles } = body;
 
-             expect(articles).toBeSortedBy("created_at",{descending:true})
+        });
+        test("200: status code responds with comments sorted by date in descending order",async () => {
+            const { status, body:{comments} } = await request(app).get("/api/articles/1/comments");
+            expect(comments).toBeSortedBy("created_at", { descending: true }); 
         })
+        test("200: status code responds with empty array when article doesn't have any comments", async () => {
+            const { status, body:{comments} } = await request(app).get("/api/articles/4/comments");
+            
+            expect(status).toBe(200);
+            expect(comments).toEqual([]);
+        });
+        test("404: status code when article not found", async () => {
+            const { status, body:{message} } = await request(app).get("/api/articles/99999/comments");
+            
+            expect(status).toBe(404);
+            expect(message).toBe("Article not found");
+        })
```

Comment:
Please add another test where the ID is not a number. That would be a bad request.